src/RoboFile/RoboFileInterface.php

```diff
@@ -43,4 +43,26 @@ public function runTestSuites(
 	 * @since   3.7.0
 	 */
 	public function prepareTestingPackage($params = array('dev' => false));
+	/**
+	 * Function for actual execution of a individual test of this extension
+	 *
+	 * @param   array  $opts  Array of configuration options:
+	 *                        - 'env': set a specific environment to get configuration from
+	 *                        - 'debug': executes codeception tasks with extended debug
+	 * 						  - 'suite':
+	 * 						  - 'test': example: 'install/InstallWeblinksCest.php/installWeblinks'
+	 *
+	 * @return void
+	 *
+	 * @since   x.x.x
+	 */
+	public function runTestSingle(
```

Comment:
This function should not be part of the interface but actually a function in the ClientContainer class, because it has already all the parameters it needs to run an actual test, so no need to re-declare on every extension.